flux/stdlib/influxdata/influxdb/rules.go

```diff
@@ -5,23 +5,55 @@ import (
 
 	"github.com/influxdata/flux"
 	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/codes"
 	"github.com/influxdata/flux/execute"
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/stdlib/influxdata/influxdb"
 	"github.com/influxdata/flux/stdlib/universe"
 	"github.com/influxdata/influxdb/models"
 	"github.com/influxdata/influxdb/storage/reads/datatypes"
 )
 
 func init() {
```

Comment:
I think we'll want/need to move this registration logic out of `init()` and into some other function that only OSS calls. Plutonium will still have access to the OSS rules that make sense for it, but it'll be able to omit any rules that need rewritten to be cluster-aware